.pre-commit-config.yaml

```
@@ -46,12 +46,17 @@ repos:
         additional_dependencies:
           - stylelint@14.2.0
           - '@wagtail/stylelint-config-wagtail@0.3.2'
+  - repo: https://github.com/thibaudcolas/curlylint
+    rev: v0.13.1
+    hooks:
+      - id: curlylint
+        args: ['--parse-only', 'wagtail']
```

Sorry for the incorrect pointer earlier, I just remembered that we shouldn't pass `wagtail` here so that the hook only runs on the staged files instead of the whole `wagtail` directory.

```suggestion
        args: ['--parse-only']
```